.github/workflows/ci.yml

```diff
@@ -100,12 +150,12 @@ jobs:
           EXT=sh
         fi
         if [[ "$OS_NAME" == "Windows" ]]; then
-          source /c/Miniconda3/Scripts/activate
-          bash build_miniforge_win.sh
+          source /c/Miniconda3/Scripts/activate;
+          source build_miniforge_win.sh;
```

Comment:
why the semicolons.

Wy dWhy do you need to source?

Reply:
We need to source because somehow, on windows, `bash foo.sh` does not carry over environment variables while it does on linux and OS X.

We should do the same in `build_miniforge_win.sh`, but somehow, this makes the constructor command fail.